### ISSUE TITLE
Accessibility: make aria-live attribute "assertive" also for 'success' and 'info' toasts

### ISSUE THREAD

**Issue by mathildew** [CodeSeven/toastr#662]
Since aria-live attribute for success and info toasts is "polite", sometimes, the reader doesn't read out loud the notification. 
They should be made assertive, since knowing that something succeeded is equally important as knowing that something failed.
